test/fixtures/userStatus/userStatus.js

```diff
@@ -86,6 +86,25 @@ const generateUserStatusData = (state, updatedAt, from, until = "", message = ""
   };
 };
 
+const generateStatusDataForCancelOOO = (userId, state) => {
+  const now = new Date();
+  let until = "";
+  const nowTimeStamp = new Date().setUTCHours(0, 0, 0, 0);
+  const fiveDaysFromNowTimeStamp = new Date(now.setUTCHours(0, 0, 0, 0) + 5 * 24 * 60 * 60 * 1000);
+  if (state === userState.OOO) {
+    until = fiveDaysFromNowTimeStamp;
+  }
+  return {
+    userId,
+    currentStatus: {
+      message: "",
```

Comment:
whats the purpose of this message field?

Reply:
The message field is when the user is OOO they can mention the reason for being OOO.